superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx

```diff
@@ -32,6 +32,7 @@ import ResizableContainer from 'src/dashboard/components/resizable/ResizableCont
 import getChartAndLabelComponentIdFromPath from 'src/dashboard/util/getChartAndLabelComponentIdFromPath';
 import useFilterFocusHighlightStyles from 'src/dashboard/util/useFilterFocusHighlightStyles';
 import { COLUMN_TYPE, ROW_TYPE } from 'src/dashboard/util/componentTypes';
+import { useTheme } from '@superset-ui/core';
```

Comment:
Ahh, you can add this import to line 25, so you're not importing stuff from superset-ui/core twice (it's a listing error). Sorry, I should have been clearer. Also, sorry you even have to DO this stuff... but it's all stuff that sets things up better for the next person :)